### PR TITLE
gracefully handle when a like is already deleted again

### PR DESCRIPTION
**Pull request by SuperTux88**
also:
- fail when sender or recipient is not found (should never happen)
- better logging

### CODE REVIEW

**Comment by jhass** [comment]
Could you try to add a test please?

**Comment by SuperTux88** [comment]
tests added

File: app/mailers/notification_mailers/base.rb [hunk 1]
```diff
@@ -29,11 +29,12 @@ def name_and_address(name, email)
...
     private
+
     def default_headers
       headers = {
-        :from => AppConfig.mail.sender_address.get,
-        :host => "#{AppConfig.pod_uri.host}",
-        :to => name_and_address(@recipient.name, @recipient.email)
+        from: AppConfig.mail.sender_address.get,
+        host: "#{AppConfig.pod_uri.host}",
+        to:   name_and_address(@recipient.name, @recipient.email)
```
Comment (jhass) [inline app/mailers/notification_mailers/base.rb]: @salbertson Hound seems to ignore our [rubocop.yml](https://github.com/diaspora/diaspora/blob/develop/.rubocop.yml#L34) here.

**Comment by denschub** [comment]
Merged as 8531b160a6c8407749411d44ac6bf72ca40408c1, :cookie:

**Pull request closed without merging**